### PR TITLE
feat: always show project selection (DSP-1274)

### PR DESCRIPTION
**Pull request by tobiasschweizer**
resolves DSP-1274

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
@subotic I found a solution to get all projects but there is a problem with when this code is executed (after lougout).

**Comment by tobiasschweizer** [comment]
@lrosenth I think we should work on this together

File: app/public/index.html [hunk 1]
```diff
@@ -347,9 +348,29 @@
...
 		}
 
-	};
+
+	}
+
+	/**
+	 * Gets all projects for selection in simple search field.
+	 */
+	function limit_simple_search_get_all_projects() {
```
Comment (tobiasschweizer) [inline app/public/index.html:363]: @lrosenth @subotic I could make this a function so it can also be called on logout.

Comment (subotic) [reply app/public/index.html:363]: If it is not much work, then that would be great.

**Review by subotic — APPROVED** [review]

File: app/public/index.html (hunk 1 above)
Comment (tobiasschweizer) [reply app/public/index.html:363]: sorry, this was misleading. I intended to say that I was able to do this :-)

**Comment by tobiasschweizer** [comment]
@subotic I'd like to add a test for this. However, #3 would be needed for that. Could you help me fix the problem that `make run` does not return and blocks github-ci?

**Pull request merged**